docs/source/how-to/estimate_how_many_cameras_can_run.rst

```diff
@@ -0,0 +1,113 @@
+   - ``"motion_vectors"``: An image containing the motion vector data at each pixel.
+   - ``"semantic_segmentation"``: The semantic segmentation data.
+   - ``"instance_segmentation_fast"``: The instance segmentation data.
+   - ``"instance_id_segmentation_fast"``: The instance id segmentation data.
+
+
+WARNING: If the ``distance_to_camera`` replicator is used to generate depth images, then they should be
+converted with ``convert_perspective_depth_image_to_orthogonal_depth_image`` from ``isaac.lab.utils.math``
+prior to creating any Point Cloud, as the ``unproject_depth`` from ``isaac.lab.utils.math`` currently assumes
+orthogonal depth.
+
+Compare Camera Type Output Through Visualization
+------------------------------------------------
+It is possible to visualize the result of the camera(s) replicators. However,
+when visualizing, the benchmark results are not meaningful due to rendering
+and the GPU to CPU conversion. Regardless, this provides a way to easily and quickly sanity check
+camera outputs.
+
+.. code-block:: bash
+
+   ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py \
+   --height 100 --width 100 --num_tiled_cameras 1 --num_standard_cameras 1 \
+   --num_ray_caster_cameras 1 --visualize
+
+This should save several images in the same directory that the script is run, showing the labelled output
+of the cameras. If depth is enabled, point clouds are generated from depth, and the rendered point cloud
+result is saved as an image as well. After the simulations stops it can be closed with CTRL C.
+
+Compare Camera Type and Performance Under Different Parameters By Benchmarking
+------------------------------------------------------------------------------
+
+If one doesn't supply the ``--visualize`` flag, then ``benchmark_cameras.py``
+can be used to estimate average rendering time for the number of desired cameras.
+
+Currently, tiled cameras are the most performant camera that can handle multiple dynamic objects.
+
+For example, to see how a system can handle 1000 tiled cameras, that are only in depth mode,
+with 1000 objects, try
+
+.. code-block:: bash
+
+   ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py \
+   --height 100 --width 100 --num_tiled_cameras 1000 --num_standard_cameras 0 \
+   --num_ray_caster_cameras 0 --tiled_camera_replicators depth --num_objects 1000
+
+If your system cannot handle this due to performance reasons, then the process will be killed.
+It's recommended to monitor CPU/RAM utilization and GPU utilization while running this script, to get
+an idea of how many resources rendering the desired camera requires. In Ubuntu, you can use tools like ``htop`` and ``nvtop``
+to live monitor resources while running this script, and in Windows, you can use the Task Manager.
+
+If your system has a hard time handling the desired cameras, you can try the following
+   - Switch to headless mode (supply ``--headless``)
```

Comment:
Please build the docs and check everything looks correct. From what I see, this bullet list might not be rendered properly.

Reply:
![image](https://github.com/user-attachments/assets/51a6a2e2-62ca-439c-81c8-1d4fc93b9b73)

Reply:
You have both a comment and a bullet list. The tabbing in there made it like that.